src/geometry/LineUtil.js

```diff
@@ -240,3 +241,19 @@ export function _flat(latlngs) {
 	console.warn('Deprecated use of _flat, please use L.LineUtil.isFlat instead.');
 	return isFlat(latlngs);
 }
+
+// @function cloneLatLngs(latlngs: LatLng[]): LatLng[]
+// Recursively clone latlngs to a new array and breaks the reference to the original one.
+export function cloneLatLngs(latlngs) {

```

Comment:
Are you sure that we need to make this function public?
@mourner 
@IvanSanchez

Reply:
It's no longer used as per review below, so let's remove it for now.

Reply:
Ok sad, because I would like to have it the Utils function. Already needed it in some projects. But before this blocks this longer I will remove it

Reply:
Let's readd it in another PR then!